sherlock.py

```diff
@@ -193,10 +241,10 @@ def sherlock(username, site_data, verbose=False, tor=False, unique_tor=False):
 
         # Retrieve future and ensure it has finished
         future = net_info["request_future"]
-        r, error_type = get_response(request_future=future,
-                                     error_type=error_type,
-                                     social_network=social_network,
-                                     verbose=verbose)
+        r, error_type, response_time = get_response(request_future=future,
```

Comment:
I think `get_response()` function takes and returns way too much data. May be we need to use DTO in future?

Reply:
Agree that it does return too much data, but didn't want to change the codebase too much in this PR.

I'm not an expert at python, is using DTOs a common strategy for solving this issue? I know it is in Java.

Reply:
It's a common strategy to use OOP :)
I've tried to refactor all code (#98) but it cannot be merged now, thus I think this function will be refactored after that. PS I like your response time metrics feature!